tools/_release.sh

```diff
@@ -264,6 +264,16 @@ if [ "$RELEASE_BRANCH" = candidate-"$version" ] ; then
     SetVersion "$nextversion".dev0
     letsencrypt-auto-source/build.py
     git add letsencrypt-auto-source/letsencrypt-auto
+    for pkg_dir in $SUBPKGS_NO_CERTBOT .
+    do
+      if [ -f "$pkg_dir/local-oldest-requirements.txt" ]; then
+        sed -i 's/-e acme[dev]/acme[dev]==$version/' "$pkg_dir/local-oldest-requirements.txt"
+        sed -i 's/-e acme/acme[dev]==$version/' "$pkg_dir/local-oldest-requirements.txt"
+        sed -i 's/-e .[dev]/certbot[dev]==$version/' "$pkg_dir/local-oldest-requirements.txt"
+        sed -i 's/-e ./certbot[dev]==$version/' "$pkg_dir/local-oldest-requirements.txt"
```

Comment:
I just tested this and I think you want to use double quotes here as the literal string `$version` was added to the file.
```
eff-bmw:tmp.Qvtu1bDpnM bmw$ version=0.29.0
eff-bmw:tmp.Qvtu1bDpnM bmw$ echo '$version'
$version
eff-bmw:tmp.Qvtu1bDpnM bmw$ echo "$version"
0.29.0
eff-bmw:tmp.Qvtu1bDpnM bmw$
```
Aren't shell scripts fun?

Reply:
The good news is Adrien has taken the task of rewriting everything in Python. The bad news is he wasn't planning on doing this script because it's never likely to be run on Windows. We could always do it anyway in the future though.

Reply:
The good news, it that I will do it eventually ^^

Reply:
Oh also on sed commands, dots and brackets needs to be escaped. They means something in the context of regular expressions.

Reply:
Adrien, I'm impressed. I didn't even mention you with @ and you still responded within 2 minutes at 11:35PM UTC :smile:

Reply:
And luckily, the interpolation of $version comes in the replace string part, not in the pattern to match part, otherwise, the dots in the variable would also be interpretated as regular expressions: here, is the real fun with shell.

Reply:
Well, @bmw, lucky you, I was checking my PRs on Certbot. When you have a full suite of them, it happens quite often :p